### ISSUE TITLE
zoralabDEBUGGER Unresponsive UI Panel and Slightly Annoying Close Button

### ISSUE THREAD

**Issue by corygalyna** [ZORALab/Hestia#386]
# Description
```
Please provide a short description of what you have encountered below.
```
As shown below.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```

![expected-debugger](https://github.com/ZORALab/Hestia/assets/124406765/97722e04-858b-469b-89e1-a25d69224592)

Effort done so far:

1. Set `border: none !important; transform: scale(1.5) !important;` when `:is(:focus, :hover)` for the close form button.
2. Set `flex-wrap: wrap !important;` against the navigation panel.

# Current Behavior
```
Please specify the current behavior of software below.
```

![current_debugger](https://github.com/ZORALab/Hestia/assets/124406765/f05a6a8e-a5b6-44dc-a9e6-7c0ab8027833)

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by JeanShuralyov** [comment]
Ouch... This is a slip.

**Comment by hollowaykeanho** [comment]
fixed in 55302ac1ab7890f602f3208bd94bad71870cca9c

**Comment by hollowaykeanho** [comment]
Released in v1.2.1.
